### ISSUE TITLE
icon picker doesn'

### ISSUE THREAD

**Issue by h3llb3nt** [jneilliii/OctoPrint-Tasmota#192]

**Comment by h3llb3nt** [comment]
unable to use the icon picker. doesn't load when using the search and manually entering also doesn't work. only one that does is the default lightning bolt
![icon-class-picker-doesn't-work](https://user-images.githubusercontent.com/22822528/209403123-fa1e27da-8e70-4fc9-ac5f-e18d29e44380.png)

**Comment by jneilliii** [comment]
please check your browser's developer tools on the console tab when loading the page and opening the icon picker. possibly a plugin conflict, it's working fine here for me.

![image](https://user-images.githubusercontent.com/5249455/209414099-d627569f-371a-4c74-bd8e-075827619373.png)

**Comment by h3llb3nt** [comment]
![Screenshot 2022-12-24 143732](https://user-images.githubusercontent.com/22822528/209449139-28c3f444-0896-4b7a-a670-6428e97691f7.png)
tried in both Chrome and Firefox. haven't tried a different machine/OS yet.

**Comment by jneilliii** [comment]
huh, what OS are you on? there is nothing in that console that makes me think there is an error.

**Comment by h3llb3nt** [comment]
windows 10, nothing fancy. i have a mac to try it on later. just tested again with my adblocker disabled. slightly different console this time
![Screenshot 2022-12-24 144452](https://user-images.githubusercontent.com/22822528/209449308-7cdcfe35-dcc5-4508-a869-88064f4ca3a6.png)

**Comment by jneilliii** [comment]
Yeah, still no errors related to my plugin. Please share your octoprint.log.

**Comment by h3llb3nt** [comment]
not alot in here since this is a brand new install im migrating to.
[octoprint.log](https://github.com/jneilliii/OctoPrint-Tasmota/files/10299298/octoprint.log)

**Comment by jneilliii** [comment]
Do me a favor and disable OctoDash Companion and see if the issue persists.

**Comment by h3llb3nt** [comment]
And that worked! Disabled. Restarted. And now the icon picker works.

**Comment by jneilliii** [comment]
Thanks, thought one of my other implementations might be interfering. OctoDash Companion plugin uses a different set of options, so it was probably overwriting this plugin's picker. I'll see what I can do to avoid that either in this plugin or that one once I have some time to investigate further.
